### PR TITLE
doc/dev/developer_guide/index.rst: add youtube references

### PR DESCRIPTION
**Pull request by neha-ojha**
Signed-off-by: Neha Ojha <nojha@redhat.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: doc/dev/developer_guide/index.rst [hunk 1]
```diff
@@ -21,10 +21,13 @@ it should serve as a reference for Ceph developers.
...
 object store and file system designed to provide excellent performance,
 reliability and scalability). If not, please refer to the `project website`_
-and especially the `publications list`_.
+and especially the `publications list`_. Another way to learn about what's happening
+in Ceph is to check out our `youtube channel`_ , where we post Tech Talks, Code walk-throughs
+and Ceph Developer Monthly recordings.
 
 .. _`project website`: https://ceph.com
 .. _`publications list`: https://ceph.com/publications/
+.. _`youtube channel`: https://www.youtube.com/channel/UCno-Fry25FJ7B4RycCxOtfw
```
Comment (joscollin) [inline doc/dev/developer_guide/index.rst]: https://support.google.com/youtube/answer/2657968?hl=en

something like: https://www.youtube.com/channel/ceph ?

**Comment by neha-ojha** [comment]
jenkins render docs

**Comment by neha-ojha** [comment]
http://docs.ceph.com/ceph-prs/29033/dev/developer_guide/#introduction

File: doc/dev/developer_guide/index.rst (hunk 1 above)
Comment (neha-ojha) [reply doc/dev/developer_guide/index.rst]: this might be a thing for Mike Perez to look at, don't think it should block this PR.

**Pull request merged**